.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+* @SebaLukas @a-w50 @barsnick @chausGit
```

Comment:
You might want to be more specific by assigning certain directories and/or file (types) to certain reviewers in the future. As a starting point this is fine

Reply:
Yes, when the time comes, I will do that :+1: